homeassistant/components/anthemav/media_player.py

```diff
@@ -55,9 +56,10 @@
 
     _LOGGER.info("Provisioning Anthem AVR device at %s:%d", host, port)
 
+    @callback
     def async_anthemav_update_callback(message):
         """Receive notification from transport that new data exists."""
-        _LOGGER.info("Received update callback from AVR: %s", message)
+        _LOGGER.debug("Received update callback from AVR: %s", message)
         hass.async_create_task(device.async_update_ha_state())
```

Comment:
`device.async_schedule_update_ha_state()`?

Reply:
I don't have the bandwidth to do cleanups in these integrations. This is merely to get the asyncs and the callbacks cleaned up.